src/main/java/Duke.java

```diff
@@ -1,69 +1,164 @@
-                index = index - 1;
+                //Extracts the index number from the text and changes status of the task
+                int index = getIndex(input);
                 lists[index].markAsDone();
                 String taskStatus = lists[index].getStatusIcon();
 
-                System.out.println(line);
+                printLineSeparator();
                 System.out.println("Nice! I've marked this task as done:");
-                System.out.println("[" + taskStatus + "] " + lists[index].getDescription());
-                System.out.println(line);
-            } else {
-                System.out.println(line);
-                System.out.println("Added: " + input);
+                System.out.println(lists[index]);
+                printLineSeparator();
+            } else if (command.equals("todo")) {
+
+                //Extracts the description and stores the task in the list
+                Task todo = getTodo(input);
 
-                //Object created for each task
-                Task t = new Task(input);
-                lists[count] = t;
-                count++;
+                printLineSeparator();
+                System.out.println("Got it. I've added this task:\n" + todo + "\nNow you have " + count + " tasks in the list.");
+                printLineSeparator();
+            } else if (command.equals("deadline")) {
 
-                System.out.println(line);
+                //Extracts the description and day/date and stores the task in the list
+                Task deadline = getDeadline(input);
+
+                printLineSeparator();
+                System.out.println("Got it. I've added this task:\n" + deadline + "\nNow you have " + count + " tasks in the list.");
+                printLineSeparator();
+            } else if (command.equals("event")) {
+
+                //Extracts the description and the time and stores the task in the list
+                Task event = getEvent(input);
+
+                printLineSeparator();
+                System.out.println("Got it. I've added this task:\n" + event + "\nNow you have " + count + " tasks in the list.");
+                printLineSeparator();
+            } else {
+                printLineSeparator();
+                System.out.println("Invalid command: " + input);
+                System.out.println("Looks like you have to try again");
+
+                printLineSeparator();
             }
-            input = in.nextLine();
         }
+    }
 
+    private static String getUserInput() {
+        String input;
+        Scanner in = new Scanner(System.in);
+        input = in.nextLine();
+
+        return input;
+    }
+
+    private static int getIndex(String input) {
+        String[] temp = input.split(" ");
+        int index = Integer.parseInt(temp[1]);
+        index = index - 1;
+
+        return index;
+    }
+
+    private static Task getTodo(String input) {
+        String todoDescription = input.substring(4).trim();
```

Comment:
can explain a bit on what is the number 4

Reply:
Thank you for pointing it out, will add a comment to it. 😃